bin/load_genotyping_plates.pl

```diff
@@ -319,29 +309,43 @@ =head2 AUTHORS
 
 my $coderef= sub  {
 
+    my $trial = CXGN::Trial->new( { bcs_schema => $schema, trial_id => $genotyping_project_id });
+    my $location_data = $trial->get_location();
+    my $location_id = $location_data->[0];
+    my $location_name = $location_data->[1];
+    my $description = $trial->get_description();
+    my $genotyping_facility = $trial->get_genotyping_facility();
+    my $plate_year = $trial->get_year();
+    
+    my $program_object = CXGN::BreedersToolbox::Projects->new( { schema => $schema });
+    my $breeding_program_data = $program_object->get_breeding_programs_by_trial($genotyping_project_id);
+    my $breeding_program_id = $breeding_program_data->[0]->[0];
```

Comment:
$breeding_program_id can be removed (using only $breeding_program_name)

Reply:
done